Editor/LyumaAv3EditorSupport.cs

```diff
@@ -269,8 +270,26 @@ static LyumaAv3EditorSupport()
 			EditorApplication.playModeStateChanged += OnPlayModeStateChange;
 			LyumaAv3Runtime.InvokeOnPreProcessAvatar = (obj) =>
 			{
+				List<IVRCSDKPreprocessAvatarCallback> _preprocessAvatarCallbacks = (List<IVRCSDKPreprocessAvatarCallback>) typeof(VRCBuildPipelineCallbacks)
+					.GetField("_preprocessAvatarCallbacks", BindingFlags.Static | BindingFlags.NonPublic).GetValue(null);
```

Comment:
Since we're accessing private data structures from the SDK, I'd like us to add null checks at each step here: between the .GetField and the .GetValue; and also the result of GetValue()

I'm worried about VRChat changing something in the future and causing it to error every time we enter play mode.